packages/server/controllers/events.js

```diff
@@ -121,6 +121,29 @@ async function getEventAttendees(eventId) {
   return query;
 }
 
+async function updateEventAttendeeStatus(userId, eventId, status) {
+  // Validate status
+  const allowedStatusValues = ["NotInterested", "Interested", "Attending"];
+  if (!allowedStatusValues.includes(status)) {
+    throw new Error(
+      "Invalid status value. Allowed values are: NotInterested, Interested, Attending"
+    );
+  }
```

Comment:
You should use an express-validator to validate the `status` request body parameter instead of doing it here. It's best to keep functions small so that they only have one responsibility. It's better to keep code small, modular, and delegate responsibilities. `updateEventAttendeeStatus` will only worry about executing the Prisma query to update the status while the express-validator will handle validation of the `status` request body input parameter.